src/pages/settings/profile/index.tsx

```diff
@@ -191,4 +306,36 @@ const SvgIcon = styled.span`
   }
 `;
 
+const NicknameMessageWrapper = styled.div<MessageWrapperProps>`
+  position: sticky;
+  width: 165px;
+  margin-top: 5px;
+
+  color: ${(props) => (props.isValid ? 'var(--font-color-primary)' : 'var(--font-color-warn)')};
+  font-size: var(--font-size-sm);
+`;
+
+const IntroduceMessageWrapper = styled.div<MessageWrapperProps>`
+  position: sticky;
+  width: 165px;
+  margin-top: 5px;
+
+  color: ${(props) => (props.isValid ? 'var(--font-color-primary)' : 'var(--font-color-warn)')};
+  font-size: var(--font-size-sm);
+`;
+
+const Button = styled.button<{ disabled: boolean }>`
+  width: 70px;
+  height: 30px;
+  margin-top: 15px;
+
+  background: ${(props) =>
+    props.disabled ? 'var(--button-color-disabled)' : 'var(--button-color-primary)'};
```

Comment:
가상선택자를 활용하면 Props를 넘겨주지 않아도 스타일을 지정할 수 잇습니다
 
```suggestion
  background: var(--button-color-primary);

  &:disabled: {
    background: var(--button-color-disabled);
  }
```